### DISCUSSION TITLE
is diagram in guide of architecture design too old?

### DISCUSSION THREAD

**Discussion by bambooHouse** [apache/dolphinscheduler#9654] [Q&A]
hi there, i'm new for dolphinscheduler,  and 
i want to study how dolphinscheduler work from guide,but i can't find MasterSchedulerThread anywhere in github.
then i found 
_https://dolphinscheduler.apache.org/zh-cn/docs/2.0.5/user_doc/architecture/design.html_
is newer than 
_https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/architecture/design.html_

correct me if i'm wrong, thank you!

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/20639249/164422825-146c29db-d332-4bf7-96c8-55e5c6e53fb5.png">
<img width="1422" alt="image" src="https://user-images.githubusercontent.com/20639249/164423137-d1cd8ce6-8be7-4231-a70c-1ae361bf1784.png">

**Comment by SnowMoon-Dev** [comment]
It is recommended to read the master source code for help

**Reply by bambooHouse** [reply]
> It is recommended to read the master source code for help

but dev branch is marked as dev, so i think it's newest.
i think we should update 
https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/architecture/design.html
so it can reflects the latest design idea, doesn't it?
